src/main/java/uk/gov/hmcts/darts/datamanagement/api/DataManagementApi.java

```diff
@@ -13,6 +13,8 @@ public interface DataManagementApi extends BlobContainerDownloadable {
 
     BinaryData getBlobDataFromUnstructuredContainer(UUID blobId);
 
+    BinaryData getBlobDataFromDetsContainer(UUID blobId);
```

Comment:
If this is DETS related wouldnt thew new method be on the uk.gov.hmcts.darts.dets.api.DetsDataManagementApi